src/main/resources/application.conf

```diff
@@ -8,6 +8,7 @@ datastax-java-driver {
   }
   advanced.metadata {
     schema.request-timeout = 10 seconds
+    token-map.enabled = false
```

Comment:
If `.conf` format allows comments, it'd be good to add a comment here explaining why this is done (copy-paste from description on this PR)?

Reply:
Done.